src/models/user-game-list-response.model.ts

```diff
@@ -0,0 +1,26 @@
+
+export interface GameListResponseGame {
+  __typename: "GameLibraryTitle";
+  conceptId: string;
+  entitlementId: string|null;
+  image: {
+    __typename: "Media";
+    url: string
+  };
+  isActive: boolean|null;
+  lastPlayedDateTime: string;
+  name: string;
+  platform: "PS4"|"PS5";
```

Comment:
Is this the same thing as `TitlePlatform`? I ask because, if so, it's possible for this `platform` value to have more potential values.

Reply:
This GraphQL API doesn't *appear* to show anything but PS4 and PS5 games, but I could be wrong. No harm in using `TitlePlatform` though 👍